kolibri/core/assets/src/api-resource.js

```diff
@@ -552,32 +574,35 @@ export class Resource {
     );
   }
 
+  cacheName(endpointName) {
+    return endpointName ? `endpoint-${endpointName}` : 'default';
+  }
+
+  getCache(type, endpointName) {
```

Comment:
A lot of the methods in these classes look internal - could we prefix them with underscores to make clear that they're not part of the public API?

Reply:
Definitely